### ISSUE TITLE
disable press-hold gesture

### ISSUE THREAD

**Issue by sean-e** [sean-e/mTroll#3]
see if disabling the press-hold gesture improves experience on touch screens.
https://blogs.msdn.microsoft.com/oldnewthing/20170227-00/?p=95585

```c
GESTURECONFIG config;
config.dwID = 0;
config.dwWant = 0;
config.dwBlock = GC_ALLGESTURES;
SetGestureConfig(hwnd, 0, 1, &config, sizeof(config));
```

**Comment by sean-e** [comment]
maybe unnecessary since `UnregisterTouchWindow` is already being used.

**Comment by sean-e** [comment]
implemented in 4b0fd43aa93542652cc7f85ec42782c268c7dd1c
